Jayce level

Fowlerpro opened the pull request:
add Hanger level and Title Screen

MohawkRaphaelT commented:
In case my Slack message gets missed, please remove the video file from the commit and instead post a public link to the video in MyCanvas.

CiaranMcIlvaney committed:
Fresh fork start (Add old scripts) Also changed movement to fixedupdate (hopefully less jittery feeling)

CiaranMcIlvaney committed:
Clean start

Changed movement in old fork by accident smh

MichaelG-design committed:
Adding Assets

I added two folders of assets offered by the profs

MichaelG-design committed:
Merge pull request #1 from brendanmcnally22/origin/Michaels-work

Adding Assets

MichaelG-design committed:
Ui Setup

Set up the Ui elements to later be attached to code
-Equiped frisbee status/inventory
-Frisbee alert
-Cooldown bar

MichaelG-design committed:
Merge pull request #2 from brendanmcnally22/origin/Michaels-work

Ui Setup

brendanmcnally22 committed:
Add Code
- Add Catch Mechanic
- Edits in Playercontroller & Frisbee.cs

brendanmcnally22 committed:
Merge pull request #3 from brendanmcnally22/feature/brendan-work

Add Code

MichaelG-design committed:
Updated UI

-Added numeral countdown visual
-Added warning light when disk goes off screen

MichaelG-design committed:
Merge pull request #4 from brendanmcnally22/origin/Michaels-work

Updated UI

MichaelG-design committed:
Imported More Assets

Added one more asset package, upon Jayce's request

MichaelG-design committed:
Merge pull request #5 from brendanmcnally22/origin/Michaels-work

Imported More Assets

Fowlerpro committed:
add Hanger level (Broken Movement)

Fowlerpro committed:
add animations with timing

Fowlerpro committed:
add Level1 Complete Animations

Fowlerpro committed:
add title screen level Change
